pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,26 @@
+{stdenv, fetchurl, which, go-md2man}:
+
+let
+  v = "1.4.3";
+in
+stdenv.mkDerivation {
+  version = v;
+  name = "brillo-${v}";
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitLab` instead. IIRC, the straightforward urls do not have a stable hash.

Reply:
Didn't know about this function, will change now